CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.6.5
+
+- Fixes issue where groups filters were not rendered correctly in metadata on Initiative page
```

Comment:
Sigh look what happens when I'm gone
```suggestion
- Fix issue where groups filters are not rendered correctly in metadata on Initiative page
```

Please apply tense fix to the 3 changelog entries below this as well